packages/datadog-instrumentations/src/aerospike.js

```diff
@@ -0,0 +1,88 @@
+'use strict'
+
+const {
+  channel,
+  addHook
+} = require('./helpers/instrument')
+const { NODE_MAJOR } = require('../../../version')
+const shimmer = require('../../datadog-shimmer')
+
+const startCh = channel('apm:aerospike:command:start')
+const endCh = channel('apm:aerospike:command:end')
+const errorCh = channel('apm:aerospike:command:error')
+
+const asyncStartChannel = channel('apm:aerospike:command:asyncStart')
+const asyncEndChannel = channel('apm:aerospike:command:asyncEnd')
+
+function wrapCreateCommand (createCommand) {
+  if (typeof createCommand !== 'function') return createCommand
+
+  return function commandWithTrace () {
+    const CommandClass = createCommand.apply(this, arguments)
+
+    if (!CommandClass) return CommandClass
+
+    shimmer.wrap(CommandClass.prototype, 'process', wrapProcess)
+
+    return CommandClass
+  }
+}
+
+function wrapProcess (process) {
+  return function (...args) {
+    if (typeof args[0] !== 'function') return process.apply(this, args)
+
+    const ctx = {
+      commandName: this.constructor.name,
+      commandArgs: this.args,
+      clientConfig: this.client.config
+    }
+
+    return startCh.runStores(ctx, () => {
+      arguments[0] = function () {
+        if (arguments[0] !== null) {
+          ctx.error = arguments['0']
```

Comment:
Is there a reason to not just add actual parameters so you can have a more descriptive reference name here?

```suggestion
          ctx.error = error
```